Shared/ThresholdKeyView.swift

```diff
@@ -468,25 +517,32 @@ struct ThresholdKeyView: View {
                         Spacer()
                         Button(action: {
                             Task {
-                                let shareStore = try! await threshold_key.generate_new_share()
-                                let index = shareStore.hex
+                                do {
+                                    let shareStore = try await threshold_key.generate_new_share()
+                                    let index = shareStore.hex
 
-                                let key_details = try! threshold_key.get_key_details()
-                                totalShares = Int(key_details.total_shares)
-                                threshold = Int(key_details.threshold)
-                                shareIndexCreated = index
+                                    let key_details = try threshold_key.get_key_details()
+                                    totalShares = Int(key_details.total_shares)
+                                    threshold = Int(key_details.threshold)
+                                    shareIndexCreated = index
 
-                                let shareOut = try! threshold_key.output_share(shareIndex: index, shareType: nil)
+                                    let shareOut = try threshold_key.output_share(shareIndex: index, shareType: nil)
 
-                                let result = try! ShareSerializationModule.serialize_share(threshold_key: threshold_key, share: shareOut, format: nil)
-                                alertContent = "serialize result is \(result)"
-                                showAlert = true
+                                    let result = try ShareSerializationModule.serialize_share(threshold_key: threshold_key, share: shareOut, format: nil)
```

Comment:
Hmm, this would be the same value I think.

With the absence of shareType and format it should produce hex in both cases.

There methods could also be changes to have the two above parameters have default values.

Reply:
Let's handle this at the another PR

Reply:
Ok, then you wouldn't need to call serialize_share here for this PR.

Reply:
what I changed in this PR was just changing try! to try.
no additional serialize_share call.